source/module_hamilt_lcao/hamilt_lcaodft/FORCE_gamma.cpp

```diff
@@ -260,15 +260,15 @@ void Force_LCAO<double>::ftable(const bool isforce,
 
         if (PARAM.inp.deepks_out_unittest)
         {
-            LCAO_deepks_io::print_dm(dm_gamma[0], PARAM.globalv.nlocal, this->ParaV->nrow);
+            LCAO_deepks_io::print_dm(1, PARAM.globalv.nlocal, this->ParaV->nrow, dm_gamma); // 1 for gamma-only
```

Comment:
rename the variable of '1', and put the variable here, don't use '1' without explanations

Reply:
Here 1 is actually nks in gamma only case, I will change it later.